theme.json

```diff
@@ -218,10 +218,16 @@
 			],
 			"fontSizes": [
 				{
+					"fluid": {
+						"min": "14px"
+					},
 					"size": "1rem",
 					"slug": "small"
 				},
 				{
+					"fluid": {
+						"min": "16px"
```

Comment:
should the units be rems?

Reply:
Yeah this would be better! Not sure why I defaulted to px 😄